docs/installation/expansion.rst

```diff
@@ -19,20 +19,14 @@ from a single entrypoint. This operation can be done either through
 
 Defining an Architecture
 ------------------------
-See the schema defined in
-:ref:`the introduction <installation-intro-install-plan>`.
-
-Once the Bootstrap is deployed, the other nodes will undergo four deployment
-runs, two for control plane nodes (bringing up the
-control plane to a total of three members), and two more for workload plane
-nodes.
-
-.. todo::
-
-   - explain architecture: 3 control plane + etcd, 2 workers (one being
-     dedicated for infra)
-   - remind roles and taints from intro
-
+Follow the recommendations provided in
+:ref:`the introduction<installation-intro-architecture>` to choose an
+architecture.
+
+List the machines to deploy and their associated roles, and deploy each of them
+using the following process, either from the GUI or CLI. Note however, that
```

Comment:
Could be nice to make `GUI or CLI` HTTP links such that users can click to follow it on another tab